developer-docs-site/docs/sdks/transactions-with-ts-sdk.md

```diff
@@ -1,172 +0,0 @@
-
-The `ModuleBundle` payload is used to publish multiple modules at once. Within `ModuleBundle` payload, you can provide the module bytecode.
-
-To transfer coins from Alice’s account to Bob’s account, we need to prepare a `ScriptFunction` payload with a `transfer` function.
-
-```ts
-// We need to pass a token type to the `transfer` function.
-const token = new TxnBuilderTypes.TypeTagStruct(TxnBuilderTypes.StructTag.fromString("0x1::aptos_coin::AptosCoin"));
-
-const entryFunctionPayload = new TxnBuilderTypes.TransactionPayloadEntryFunction(
-  TxnBuilderTypes.EntryFunction.natural(
-    // Fully qualified module name, `AccountAddress::ModuleName`
-    "0x1::coin",
-    // Module function
-    "transfer",
-    // The coin type to transfer
-    [token],
-    // Arguments for function `transfer`: receiver account address and amount to transfer
-    [BCS.bcsToBytes(TxnBuilderTypes.AccountAddress.fromHex(bob.address())), BCS.bcsSerializeUint64(717)],
-  ),
-);
-```
-
-The Move function `transfer` requires a coin type as type argument. The function `transfer` is defined here [https://github.com/aptos-labs/aptos-core/blob/faf4f94260d4716c8a774b3c17f579d203cc4013/aptos-move/framework/aptos-framework/sources/Coin.move#L311](https://github.com/aptos-labs/aptos-core/blob/faf4f94260d4716c8a774b3c17f579d203cc4013/aptos-move/framework/aptos-framework/sources/Coin.move#L311).
-
-In above code snippet, we want to transfer the `AptosCoin` that is defined under account `0x1` and module `AptosCoin`. The fully qualified name for the `AptosCoin` is therefore `0x1::aptos_coin::AptosCoin`.
-
-:::note
-All arguments in `ScriptFunction` payload must be BCS serialized. In above code, we serialized Bob’s account address and the amount number to transfer.
-:::
-
-## Step 3: Sign and submit the transaction
-
-After assembling a transaction payload, we are ready to create a `RawTransaction` instance that wraps the payload we just created. The `RawTransaction` can then be signed and submitted.
-
-```ts
-// Create a raw transaction out of the transaction payload
-const rawTxn = await client.generateRawTransaction(alice.address(), entryFunctionPayload);
-
-// Sign the raw transaction with Alice's private key
-const bcsTxn = AptosClient.generateBCSTransaction(alice, rawTxn);
-// Submit the transaction
-const transactionRes = await client.submitSignedBCSTransaction(bcsTxn);
-
-// Wait for the transaction to finish
-await client.waitForTransaction(transactionRes.hash);
-
-resources = await client.getAccountResources(bob.address());
-accountResource = resources.find((r) => r.type === "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>");
-console.log(`Bob coins: ${(accountResource?.data as any).coin.value}. Should be 717!`);
-```
-
-## Output
-
-The output after executing:
-
-```tsx
-Alice coins: 5000. Should be 5000!
-Bob coins: 0. Should be 0!
-Bob coins: 717. Should be 717!
-```
-
-## Build raw transactions with the ABI transaction builder
```

Comment:
When I added `CoinClient` to the SDK I found this section essential. Without it I would've had no idea what to do. Could we add this to the SDK README perhaps, or even just as a comment in `abis.ts`.

Reply:
sounds good. adding to `abis.ts` makes sense.